src/@next/Bar/BarStyle.ts

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import { Device, ScreenSize } from '../..';
+import { Neutral } from '../utilities/colors';
+import { space8 } from '../utilities/spacing';
+
+export const StyledBar = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 20px 0;
+  width: 100%;
+  position: absolute;
+
+  &[data-align='bottom'] {
+    bottom: 0;
+  }
+  &[data-align='top'] {
+    top: 0;
+  }
+
+  box-shadow: 0px 0px 0px 1px rgba(63, 63, 68, 0.05),
+    0px 1px 3px rgba(63, 63, 68, 0.15);
+  background: ${Neutral.B100};
+`;
+
+export const StyledBarHeaderWrapper = styled.div`
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+`;
+
+export const StyledBarActionWrapper = styled.div`
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: flex-end;
+`;
+
+export const StyledBarContainer = styled.div`
```

Comment:
please add:
```
width: 100%;
padding: (50px left and right for large screens and 24px left and right for smaller screens)
```

to stretch the content.

from this:
![Screenshot 2023-03-08 at 2 05 35 PM](https://user-images.githubusercontent.com/7514754/223632601-5c01a129-187b-4912-a06a-c9572ed3b075.png)

to this:
![Screenshot 2023-03-08 at 2 05 12 PM](https://user-images.githubusercontent.com/7514754/223632635-0268a555-057c-4e24-8867-dcc6c5ef28a1.png)